kythe/typescript/indexer.ts

```diff
@@ -1596,8 +1597,11 @@ class Visitor {
     let declKw;
     if (ts.isVariableDeclaration(varDecl)) {
       declKw = initializerList.kind === ts.SyntaxKind.CatchClause ?
-          '(local var)' :
-          initializerList.flags & ts.NodeFlags.Const ? 'const' : 'let';
+                                                       '(local var)' :
+          initializerList.flags & ts.NodeFlags.Const ? 'const' :
+                                                       'let';
+    } else if (ts.isParameter(varDecl)) {
+      declKw = '(parameter)';
```

Comment:
BTW, do you think it is valuable to include this info in the output?  I feel like as a user of a variable I don't really care if it's a let vs a var vs a param or whatever.

Reply:
I think it will be valuable when we get to things like functions and classes, where the declaration context provides more insight than in this case. I also think it worth providing as much information as possible via marked source, and then the UI can decide what is displayed to the user.